### PR TITLE
Update ios.py

### PR DESCRIPTION
**Pull request by sinbycosmay**
Added some comments and made the file more readable and programmer friendly.

### CODE REVIEW

File: src/briefcase/ios.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def finalize_options(self):
     def install_icon(self):
         last_size = None
         for size in [
-                    '1024',  # App Store Icon
+                    '1024', # App Store Icon
```
Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 28: (E261) at least two spaces before inline comment

File: src/briefcase/ios.py [hunk 2]
```diff
@@ -165,7 +165,7 @@ def set_device_target(self):
                             self.device_name = self.device['name']
                         except:
```
Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 25: (E722) do not use bare except'

File: src/briefcase/ios.py [hunk 3]
```diff
@@ -165,7 +165,7 @@ def set_device_target(self):
...
                             print("Invalid selection.")
-                            print
+                            print #Giving space for new process to print with a more readable format
```
Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 34: (E261) at least two spaces before inline comment

Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 35: (E262) inline comment should start with '# '

File: src/briefcase/ios.py [hunk 4]
```diff
@@ -165,7 +165,8 @@ def set_device_target(self):
                             self.device_name = self.device['name']
                         except:
```
Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 25: (E722) do not use bare except'

File: src/briefcase/ios.py [hunk 5]
```diff
@@ -165,7 +165,8 @@ def set_device_target(self):
...
                             print("Invalid selection.")
-                            print
+                            print
```
Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 34: (W291) trailing whitespace

File: src/briefcase/ios.py [hunk 6]
```diff
@@ -165,7 +165,8 @@ def set_device_target(self):
                             self.device_name = self.device['name']
                         except:
                             print("Invalid selection.")
-                            print
+                            print 
+                            #Giving space for new process to print with a more readable format
```
Comment (brutusthebee) [inline src/briefcase/ios.py]: At column 29: (E265) block comment should start with '# '

File: src/briefcase/ios.py (hunk 4 above)
Comment (sinbycosmay) [reply src/briefcase/ios.py]: I believe I do not have the prerequisite knowledge to use it

**Comment by freakboy3742** [comment]
What exactly are you trying to achieve with this patch? It seems like little more than shifting some whitespace around in some comments...

**Comment by freakboy3742** [comment]
Closing this PR; I strongly suspect it was an attempt to pad number for Hacktoberfest.

**Pull request closed without merging**